src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -5,6 +5,31 @@ import { getJWTRequest, TokenType, validateQsh } from "~/src/jira/util/jwt";
 import { Installation } from "~/src/models/installation";
 import { moduleUrls } from "~/src/routes/jira/atlassian-connect/jira-atlassian-connect-get";
 import { matchRouteWithPattern } from "~/src/util/match-route-with-pattern";
+import { getJiraClient } from "~/src/jira/client/jira-client";
+
+export const setJiraAdminPrivileges = async (req: Request, claims, jiraHost, installationId) => {
+	// We only need to add this to the session if it doesn't exist
+	if (req.session.isJiraAdmin !== undefined) {
+		return;
+	}
+	try {
+		const userAccountId = claims.sub;
```

Comment:
`sub` could be empty according to https://developer.atlassian.com/cloud/jira/platform/understanding-jwt-for-connect-apps/#claims
So maybe we should assume if `accountId` is empty, then assume the action is triggered from somewhere non-user, and therefore as "admin"?
![image](https://user-images.githubusercontent.com/105693507/229957754-e2df9968-4a38-4d08-bc2f-f23c368c9cfc.png)

Reply:
if they arent logged in it will be emtpy so thats fine.

As far as missing due to accessing from another methid or some such, we can handle that when we refuse the endpoint. e.g. if(session.isJiraAdmin === undefined || developers_backdoor_muahahahha)